ci/pin-msrv.sh

```diff
@@ -28,3 +28,4 @@ cargo update -p once_cell --precise "1.20.3"
 cargo update -p base64ct --precise "1.6.0"
 cargo update -p minreq --precise "2.13.2"
 cargo update -p tracing-core --precise "0.1.33"
+cargo update -p webpki-roots@1.0.2 --precise "1.0.1"
```

Comment:
nit: I wonder if there's a way to use the cargo package specification with a matching regex, in order to not need to update once another version is released. I've tried locally without success.